spec.html

```diff
@@ -13579,17 +13579,18 @@ <h1>
           _key_: a property key or Private Name,
           _closure_: a function object,
           _enumerable_: a Boolean,
-        ): a PrivateElement or ~unused~
+        ): either a normal completion containing either a PrivateElement or ~unused~, or an abrupt completion
       </h1>
       <dl class="header">
       </dl>
       <emu-alg>
-        1. Assert: _homeObject_ is an ordinary, extensible object with no non-configurable properties.
+        1. Assert: _homeObject_ is an ordinary, extensible object.
         1. If _key_ is a Private Name, then
           1. Return PrivateElement { [[Key]]: _key_, [[Kind]]: ~method~, [[Value]]: _closure_ }.
         1. Else,
           1. Let _desc_ be the PropertyDescriptor { [[Value]]: _closure_, [[Writable]]: *true*, [[Enumerable]]: _enumerable_, [[Configurable]]: *true* }.
-          1. Perform ! DefinePropertyOrThrow(_homeObject_, _key_, _desc_).
+          1. Perform ? DefinePropertyOrThrow(_homeObject_, _key_, _desc_).
+          1. NOTE: DefinePropertyOrThrow only returns an abrupt completion when attempting to define a class static method whose _key_ is *"prototype"*.
```

Comment:
Can we move the note up to before the DefinePropertyOrThrow step? I'd prefer to read the note first.

Reply:
I prefer it after. With none of us caring that much and Kevin voting in favor of doing the minimal effort thing, let's leaving it as is.